core/src/tangram.cpp

```diff
@@ -214,9 +219,9 @@ void Map::loadSceneAsync(const char* _scenePath, bool _useScenePosition, MapRead
         nextScene = impl->nextScene;
     }
 
-    runAsyncTask([nextScene, _platformCallback, _cbData, this](){
+    runAsyncTask([nextScene, _platformCallback, _cbData, _sceneUpdates, this](){
```

Comment:
Need to check this. Based on the documentation for lambda capture this is a capture of the value!! But the original `_sceneUpdates` is a `const reference`, not sure what will happen. Will test it.